inception-recommendation/src/main/java/de/tudarmstadt/ukp/inception/recommendation/project/ProjectRecommendersPanel.java

```diff
@@ -58,6 +60,10 @@ public ProjectRecommendersPanel(String aId, IModel<Project> aProject)
             _target.add(recommenderEditorPanel);
         });
         add(recommenderListPanel);
+        
+        EvaluationSimulationPanel evaluationSimulationPanel = new EvaluationSimulationPanel(
+                EVALUATION_SIMULATION_CONTAINER, projectModel.getObject(), recommenderEditorPanel);
```

Comment:
Instead of passing the `recommenderEditorPanel` here, pass the `selectedRecommenderModel`.

Reply:
Done,